roles/rhol_crc/molecule/default/converge.yml

```diff
@@ -22,5 +22,19 @@
     cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
     cifmw_manage_secrets_pullsecret_content: >
       {"auths":{"fake":{"auth": "Zm9vOmJhcgo="}}}
-  roles:
-    - role: "rhol_crc"
+  tasks:
+    - name: Add crc hostname with it's IP to /etc/hosts
+      become: true
+      ansible.builtin.lineinfile:
+        path: /etc/hosts
+        line: "192.168.130.11 crc"
```

Comment:
not sure the hard-coded IP is right here? Do we really need that record actually?

Reply:
scratch that  it's in molecule